From 6e54973c3c758674c3d04b5e2cf12e097006f6a3 Mon Sep 17 00:00:00 2001
From: Mark Mentovai <mark@mentovai.com>
Date: Fri, 29 Sep 2023 11:03:26 -0400
Subject: [PATCH] Fix build with clang -Wunqualified-std-cast-call

This warning is enabled by default since clang 15. This fixes a build
error with those recent versions of clang because BOSSA builds with
-Werror.
---
 src/Samba.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git src/Samba.cpp src/Samba.cpp
index 490e3936a445..e2076bb6c194 100644
--- src/Samba.cpp
+++ src/Samba.cpp
@@ -141,7 +141,7 @@ Samba::init()
 bool
 Samba::connect(SerialPort::Ptr port, int bps)
 {
-    _port = move(port);
+    _port = std::move(port);
 
     // Try to connect at a high speed if USB
     _isUsb = _port->isUsb();
-- 
2.42.0

